packages/stark-ui/src/modules/session-ui/pages/session-logout/session-logout-page.component.html

```diff
@@ -0,0 +1,12 @@
+<div class="stark-session-ui-page" role="alertdialog" aria-busy="true" aria-live="assertive">
+	<header><i></i></header>
+	<content>
+		<h2 translate>STARK.SESSION_LOGOUT.TITLE</h2>
+
+		<button class="md-raised md-primary"
+				(click)="logon()"
+				[attr.aria-label]="'STARK.SESSION_LOGOUT.LOGIN' | translate">
+			<span translate>STARK.SESSION_LOGOUT.LOGIN</span>
+		</button>
+	</content>
```

Comment:
It seems this page is not correctly styled :(
![no styled logout page](https://user-images.githubusercontent.com/5698318/46015383-7269c680-c0d2-11e8-8829-e4442aa53a66.PNG)